pkg/controller/accountclaim/reuse.go

```diff
@@ -125,7 +125,6 @@ func (r *ReconcileAccountClaim) cleanUpAwsAccount(reqLogger logr.Logger, account
 		case errMsg := <-awsErrors:
 			err = errors.New(errMsg)
 			reqLogger.Error(err, errMsg)
-			return err
```

Comment:
I don't think we can simply remove this return. We still need to inform the caller function that we had an error during the cleanup. That error will trigger the account being flagged as Failed Reuse.

Maybe we could store the error messages and a flag error = true, and after all the 3 functions finish we return it?

Reply:
Ah I see yes we can just keep a bool indicating there was an error and return once the routines are complete.

Reply:
/lgtm